SearchingEngine.java

```diff
@@ -0,0 +1,167 @@
+package SearchingEngine;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+/**
+ * Description of how to use it you can find in README.md
+ * */
+
+public class SearchingEngine {
+    public static void main(String[] args) throws FileNotFoundException {
+        Scanner textFile = new Scanner(new File(findPath(args)));
+        mainMenu(allList(textFile));
```

Comment:
Regarding clean code practices, the name of the method ideally should be the action that it performs.